content/200-orm/200-prisma-client/500-deployment/301-edge/485-deploy-to-vercel.mdx

```diff
@@ -0,0 +1,458 @@
+---
+title: 'Deploy to Vercel Edge Functions & Middleware'
+navTitle: 'Deploy to Vercel'
+metaTitle: 'Deploy to Vercel Edge Functions'
+metaDescription: 'Learn the things you need to know in order to deploy an Edge function that uses Prisma Client for talking to a database.'
+tocDepth: 3
+preview: true
+---
+
+<TopBlock>
+
+This page covers everything you need to know in order to deploy an edge function to [Vercel Edge Functions](https://vercel.com/docs/functions/edge-functions) that uses Prisma Client for talking to a database.
```

Comment:
This links to a generic "Vercel Functions" page now.

Reply:
updated the intro to link to updated locations in the vercel docs 👍

Reply:
Updated sentence does not really make sense now, looks like something got messed up.

Reply:
new version:

---

This page covers everything you need to know to deploy an app that uses Prisma Client for talking to a database in [Vercel Edge Middleware](https://vercel.com/docs/functions/edge-middleware) or a [Vercel Function](https://vercel.com/docs/functions) deployed to the [Vercel Edge Runtime](https://vercel.com/docs/functions/runtimes/edge-runtime).

To deploy a Vercel Function to the Vercel Edge Runtime, you can set `export const runtime = 'edge'` outside the request handler of the Vercel Function.